layers/dynamic_rnn.py

```diff
@@ -57,10 +57,16 @@ def forward(self, x, x_len):
         :return:
         """
         """sort"""
-        x_sort_idx = torch.argsort(-x_len)
-        x_unsort_idx = torch.argsort(x_sort_idx).long()
-        x_len = x_len[x_sort_idx]
-        x = x[x_sort_idx.long()]
+        if torch.__version__.startswith('1'):
```

Comment:
Is it better to use `torch.sort` only if  `torch.__version__.startswith('0')` ?

Reply:
Yes, you're right.
And here comes another question: could we only use `torch.sort` to avoid condition judgement as `torch.sort` and `torch.argsort` are almost the same on time complexity in my trial ?

Reply:
I‘ve updated the code. 👌